python/ray/tests/test_metrics.py

```diff
@@ -377,6 +379,444 @@ def getpid(self):
     assert profiling_stats is not None
 
 
+def test_memory_dashboard(shutdown_only):
+    """Test Memory table.
+
+    These tests are motivated by this document.
+    https://docs.ray.io/en/latest/memory-management.html#debugging-using-ray-memory
+    """
+    addresses = ray.init(num_cpus=2)
+    webui_url = addresses["webui_url"].replace("localhost", "http://127.0.0.1")
+    assert (wait_until_server_available(addresses["webui_url"]) is True)
+
+    def get_memory_table():
+        memory_table = requests.get(webui_url + "/api/memory_table").json()
+        return memory_table["result"]
+
+    def memory_table_ready():
+        memory_table = get_memory_table()
+        return len(memory_table["group"]) > 0
+
+    def stop_memory_table():
+        requests.get(webui_url + "/api/stop_memory_table").json()
+        time.sleep(1.0)
```

Comment:
Generally we don't want to use `time.sleep` for tests. Because travis can take longer than 1s for whatever we are waiting here. If we want to verify the memory table is stopped, can we ping `/api/memory_table` again and see if we receive some error saying memory table has been stopped?

Reply:
Agree. This was a bit tricky because API behaviors I want is not test-friendly. I will come up with a good idea! Thanks for the feedback.